tests/ui/eq_op.rs

```diff
@@ -86,3 +86,12 @@ fn check_ignore_macro() {
     // checks if the lint ignores macros with `!` operator
     !bool_macro!(1) && !bool_macro!("");
 }
+
+struct Nested {
+    inner: ((i32,), (i32,), (i32,)),
+}
+
+fn check_nested(n1: &Nested, n2: &Nested) -> bool {
+    // There's no `n2.inner.0.0`
```

Comment:
I was confused when first reading this comment. Maybe reword it as follows:

```suggestion
    // `n2.inner.0.0` mistyped as `n1.inner.0.0`
```